SimG4Core/Application/src/Phase2SteppingAction.cc

```diff
@@ -13,6 +13,8 @@
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "FWCore/Utilities/interface/isFinite.h"
 
+#include <DD4hep/Filter.h>
```

Comment:
@bsunanda , let us somehow remove this header.

Reply:
I am working on this